tests/test_viz.py

```
@@ -8,24 +8,29 @@
 import pytest
 
 import sys
+
 sys.path.append("../")
 
-import missingno as msno
+from missingno import missingno as msno
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)